cmake/projects/xcb/hunter.cmake

```diff
@@ -46,6 +46,6 @@ hunter_pick_scheme(DEFAULT xcb)
 hunter_cacheable(xcb)
 hunter_download(
     PACKAGE_NAME xcb
-    PACKAGE_INTERNAL_DEPS_ID "3"
+    PACKAGE_INTERNAL_DEPS_ID "7"
```

Comment:
"3" -> "7" looks strange, please use "4" here

Reply:
Well, I expected that I should for every change/commit increment the package internal id to execute test builds .... So I should only increment once in a branch and for every commit in the branch, test builds are executed?